### PR TITLE
docs: more detailed explanation of opener attribute

### PR DESCRIPTION
**Pull request by MarcusNotheis**
### PR checklist
- [x] Follow the [Commit message Guidelines](https://github.com/SAP/ui5-webcomponents/blob/main/docs/6-contributing/02-conventions-and-guidelines.md#commit-message-style)
- [x] Add proper description about the background of the change and the change itself

- [x] Link to an existing issue (if available)
   Fixes: #8678

- [x] Read the [Contributing Guidelines](https://github.com/SAP/ui5-webcomponents/blob/main/CONTRIBUTING.md)

### CODE REVIEW

**Review by nnaydenow — APPROVED** [review]

File: packages/main/src/ColorPalettePopover.ts [hunk 1]
```diff
@@ -134,6 +134,8 @@ class ColorPalettePopover extends UI5Element {
 
 	/**
 	 * Defines the ID or DOM Reference of the element that the popover is shown at.
+	 * When using this attribute in a declarative way you must only use the `id` as string of the element you want to show the popover at.
```
Comment (BorisDafov) [inline packages/main/src/ColorPalettePopover.ts:137]: I suggest:
When using this attribute in a declarative way, you must only use the `id` (as a string) of the element at which you want to show the popover.

File: packages/main/src/Menu.ts [hunk 2]
```diff
@@ -253,7 +253,9 @@ class Menu extends UI5Element {
 	loadingDelay!: number;
 
 	/**
-	 * Defines the ID or DOM Reference of the element that the menu is shown at
+	 * Defines the ID or DOM Reference of the element that the menu is shown at.
```
Comment (BorisDafov) [inline packages/main/src/Menu.ts:256]: I suggest:
Defines the ID or DOM Reference of the element at which the menu is shown.

File: packages/main/src/Menu.ts [hunk 3]
```diff
@@ -253,7 +253,9 @@ class Menu extends UI5Element {
...
+	 * When using this attribute in a declarative way you must only use the `id` as string of the element you want to show the popover at.
```
Comment (BorisDafov) [inline packages/main/src/Menu.ts:257]: same:
When using this attribute in a declarative way, you must only use the `id` (as a string) of the element at which you want to show the popover.

File: packages/main/src/Popover.ts [hunk 4]
```diff
@@ -222,7 +222,9 @@ class Popover extends Popup {
 	}
 
 	/**
-	 * Defines the ID or DOM Reference of the element that the popover is shown at
+	 * Defines the ID or DOM Reference of the element that the popover is shown at.
```
Comment (BorisDafov) [inline packages/main/src/Popover.ts:225]: similar:
Defines the ID or DOM Reference of the element at which the popover is shown.
When using this attribute in a declarative way, you must only use the `id` (as a string) of the element at which you want to show the popover.

File: packages/main/src/Select.ts [hunk 5]
```diff
@@ -207,6 +207,8 @@ class Select extends UI5Element implements IFormElement {
 	/**
 	 * Defines a reference (ID or DOM element) of component's menu of options
 	 * as alternative to define the select's dropdown.
+	 * When using this attribute in a declarative way you must only use the `id` as string of the element you want to use as menu.
```
Comment (BorisDafov) [inline packages/main/src/Select.ts:210]: same:
When using this attribute in a declarative way, you must only use the `id` (as a string) of the element at which you want to show the menu.

File: packages/main/src/Tokenizer.ts [hunk 6]
```diff
@@ -257,6 +257,8 @@ class Tokenizer extends UI5Element {
 
 	/**
 	 * Defines the ID or DOM Reference of the element that the menu is shown at
+	 * When using this attribute in a declarative way you must only use the `id` as string of the element you want to show the menu at.
```
Comment (BorisDafov) [inline packages/main/src/Tokenizer.ts:260]: same:
When using this attribute in a declarative way, you must only use the `id` (as a string) of the element at which you want to show the menu.

**Review by BorisDafov — CHANGES_REQUESTED** [review]

**Comment by MarcusNotheis** [comment]
Thanks for the review, @BorisDafov. I've changed the descriptions as suggested by you.

**Pull request merged**